### ISSUE TITLE
Cant install it with pip(version 21.1.2, python version: 3.9.1)

### ISSUE THREAD

**Issue by Frosterast2** [r0x0r/pywebview#733]
There seems to be issues for people when installing pywebview with pip. These are the logs `Collecting pywebview
  Using cached pywebview-3.4-py3-none-any.whl (325 kB)
Collecting pythonnet
  Using cached pythonnet-2.5.2.tar.gz (1.9 MB)
Requirement already satisfied: pycparser in c:\users\saketh\appdata\local\programs\python\python39\lib\site-packages (from pythonnet->pywebview) (2.20)
Building wheels for collected packages: pythonnet
  Building wheel for pythonnet (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: 'c:\users\saketh\appdata\local\programs\python\python39\python.exe' -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\saketh\\AppData\\Local\\Temp\\pip-install-jbv7298e\\pythonnet_226454c7328b4cbcb870fe5f73e55f89\\setup.py'"'"'; __file__='"'"'C:\\Users\\saketh\\AppData\\Local\\Temp\\pip-install-jbv7298e\\pythonnet_226454c7328b4cbcb870fe5f73e55f89\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d 'C:\Users\saketh\AppData\Local\Temp\pip-wheel-v52wiex0'
       cwd: C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\
  Complete output (39 lines):
  running bdist_wheel
  running build
  running build_ext
  Checking for updates from https://www.nuget.org/api/v2/.
  Currently running NuGet.exe 4.1.0.
  Updating NuGet.exe to 5.9.1.
  Update successful.
  Cannot find the specified version of msbuild: '14'
  MSBuild auto-detection: using msbuild version '4.0' from 'C:\Windows\Microsoft.NET\Framework64\v4.0.30319'.
  C:\Users\saketh\AppData\Local\Temp\NuGetScratch\azeyy5zh.s4g.nugetrestore.targets(460,20): error MSB4086: A numeric comparison was attempted on "$(MSBuildAssemblyVersion)" that evaluates to "" instead of a number, in condition "'$(MSBuildAssemblyVersion)' < '15.0'". [C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\src\runtime\Python.Runtime.csproj]
  C:\Users\saketh\AppData\Local\Temp\NuGetScratch\azeyy5zh.s4g.nugetrestore.targets(460,20): error MSB4086: A numeric comparison was attempted on "$(MSBuildAssemblyVersion)" that evaluates to "" instead of a number, in condition "'$(MSBuildAssemblyVersion)' < '15.0'". [C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\src\testing\Python.Test.csproj]  
  C:\Users\saketh\AppData\Local\Temp\NuGetScratch\azeyy5zh.s4g.nugetrestore.targets(460,20): error MSB4086: A numeric comparison was attempted on "$(MSBuildAssemblyVersion)" that evaluates to "" instead of a number, in condition "'$(MSBuildAssemblyVersion)' < '15.0'". [C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\src\embed_tests\Python.EmbeddingTest.csproj]
  C:\Users\saketh\AppData\Local\Temp\NuGetScratch\azeyy5zh.s4g.nugetrestore.targets(460,20): error MSB4086: A numeric comparison was attempted on "$(MSBuildAssemblyVersion)" that evaluates to "" instead of a number, in condition "'$(MSBuildAssemblyVersion)' < '15.0'". [C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\src\console\Console.csproj]      
  C:\Users\saketh\AppData\Local\Temp\NuGetScratch\azeyy5zh.s4g.nugetrestore.targets(460,20): error MSB4086: A numeric comparison was attempted on "$(MSBuildAssemblyVersion)" that evaluates to "" instead of a number, in condition "'$(MSBuildAssemblyVersion)' < '15.0'". [C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\src\clrmodule\clrmodule.csproj]  

  WARNING: Error reading msbuild project information, ensure that your input solution or project file is valid. NETCore and UAP projects will be skipped, only packages.config files will be restored.   
  Restoring NuGet package System.ValueTuple.4.5.0.
  Restoring NuGet package UnmanagedExports.1.2.7.
  Restoring NuGet package NUnit.ConsoleRunner.3.11.1.
  Restoring NuGet package NUnit.3.12.0.
  Adding package 'NUnit.ConsoleRunner.3.11.1' to folder 'C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\packages'
  Adding package 'UnmanagedExports.1.2.7' to folder 'C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\packages'
  Adding package 'System.ValueTuple.4.5.0' to folder 'C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\packages'
  Adding package 'NUnit.3.12.0' to folder 'C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\packages'
  Added package 'UnmanagedExports.1.2.7' to folder 'C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\packages'
  Added package 'NUnit.ConsoleRunner.3.11.1' to folder 'C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\packages'
  Added package 'System.ValueTuple.4.5.0' to folder 'C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\packages'
  Added package 'NUnit.3.12.0' to folder 'C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\packages'

  NuGet Config files used:
      C:\Users\saketh\AppData\Roaming\NuGet\NuGet.Config

  Feeds used:
      C:\Users\saketh\.nuget\packages\
      https://api.nuget.org/v3/index.json

  Installed:
      4 package(s) to packages.config projects
  error: [WinError 2] The system cannot find the file specified
  ----------------------------------------
  ERROR: Failed building wheel for pythonnet
  Running setup.py clean for pythonnet
Failed to build pythonnet
Installing collected packages: pythonnet, pywebview
    Running setup.py install for pythonnet ... error
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\saketh\appdata\local\programs\python\python39\python.exe' -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\saketh\\AppData\\Local\\Temp\\pip-install-jbv7298e\\pythonnet_226454c7328b4cbcb870fe5f73e55f89\\setup.py'"'"'; __file__='"'"'C:\\Users\\saketh\\AppData\\Local\\Temp\\pip-install-jbv7298e\\pythonnet_226454c7328b4cbcb870fe5f73e55f89\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\saketh\AppData\Local\Temp\pip-record-4s2ml9o9\install-record.txt' --single-version-externally-managed --compile --install-headers 'c:\users\saketh\appdata\local\programs\python\python39\Include\pythonnet'
         cwd: C:\Users\saketh\AppData\Local\Temp\pip-install-jbv7298e\pythonnet_226454c7328b4cbcb870fe5f73e55f89\
    Complete output (6 lines):
    usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
       or: setup.py --help [cmd1 cmd2 ...]
       or: setup.py --help-commands
       or: setup.py cmd --help

    error: option --single-version-externally-managed not recognized
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'c:\users\saketh\appdata\local\programs\python\python39\python.exe' -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\saketh\\AppData\\Local\\Temp\\pip-install-jbv7298e\\pythonnet_226454c7328b4cbcb870fe5f73e55f89\\setup.py'"'"'; __file__='"'"'C:\\Users\\saketh\\AppData\\Local\\Temp\\pip-install-jbv7298e\\pythonnet_226454c7328b4cbcb870fe5f73e55f89\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\saketh\AppData\Local\Temp\pip-record-4s2ml9o9\install-record.txt' --single-version-externally-managed --compile --install-headers 'c:\users\saketh\appdata\local\programs\python\python39\Include\pythonnet' Check the logs for full command output.`

Is there any other way to install it or is there any other method

**Comment by r0x0r** [comment]
It is a pythonnet issue. Check with them what is the latest version they support.

**Comment by mishraabhi0123** [comment]
I faced this issue as well. For some reason pip was not able to get the `.whl` for the `pythonnet`. This is how I resolved it.
1. Manually download the `.whl` file for `pythonnet` from [here](https://www.lfd.uci.edu/~gohlke/pythonlibs/#pythonnet) and save it to any location of your choice. Make sure to download according to your system specification.
2. If you are using virtual environment, activate the environment. Otherwise ignore this step.
3. Now run `pip install <path to pythonnet whl file>`
